### PR TITLE
DDT-1232: Look up document references for events

### PR DESCRIPTION
**Pull request by Abdiiir**

### CODE REVIEW

File: tnt-transfer-obj/src/main/java/org/dcsa/tnt/transferobjects/RelatedDocumentReferenceTO.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package org.dcsa.tnt.transferobjects;
```
Comment (FinnAtAsseco) [inline tnt-transfer-obj/src/main/java/org/dcsa/tnt/transferobjects/RelatedDocumentReferenceTO.java:1]: (Optional) This class can probably go. We don't care much about 2.2 compatibility and in 3.0 the fields are the same in all 3 events.

**Comment by FinnAtAsseco** [comment]
DocumentReferenceType  should be included in the EnumValuesTest

**Review by FinnAtAsseco — CHANGES_REQUESTED** [review]

File: tnt-transfer-obj/src/main/java/org/dcsa/tnt/transferobjects/RelatedDocumentReferenceTO.java (hunk 1 above)
Comment (Abdiiir) [reply tnt-transfer-obj/src/main/java/org/dcsa/tnt/transferobjects/RelatedDocumentReferenceTO.java:1]: They are not the same on the 3.0 TNT documentation on swaggerhub.

Comment (FinnAtAsseco) [reply tnt-transfer-obj/src/main/java/org/dcsa/tnt/transferobjects/RelatedDocumentReferenceTO.java:1]: https://app.swaggerhub.com/apis/dcsaorg/DCSA_TNT/3.0.0-Beta-1#/Events/get_v3_events

All three event types use relateddocumentreferences with a "type" and a "value"

**Review by FinnAtAsseco — APPROVED** [review]

**Pull request merged**